node/node.go

```diff
@@ -310,16 +385,24 @@ func (nn *nexNode) Auction(auctionId string, agentType []string, tags map[string
 		return nil, err
 	}
 	for _, c := range agentSuper.Children() {
-		agentResp, err := agentSuper.Ask(nn.ctx, c, &actorproto.PingAgent{})
+		agentResp, err := agentSuper.Ask(nn.ctx, c, &actorproto.PingAgent{}, 5*time.Second)
```

Comment:
Should this be the default ask timeout?

Reply:
Again, going back to the multiple timeout shuffle.  The `agent.Ask` didn't have a timeout field a week ago, so I am trying to figure out what makes sense will the timing of everything....